src/VPolygon.jl

```diff
@@ -553,3 +553,52 @@ function translate(P::VPolygon{N}, v::AbstractVector{N}) where {N<:Real}
                                 "set by a $(length(v))-dimensional vector"
     return VPolygon([x + v for x in vertices_list(P)])
 end
+"""
+    minkowski_sum(P::VPolygon{N}, Q::VPolygon{N}) where {N<:Real}
+
+The Minkowski Sum of two polygon in vertex representation.
+
+### Input
+
+- `P` -- polygon in vertex representation
+- `Q` -- another polygon in vertex representation
+
+### Output
+
+A polygon in vertex representation.
+
+### Algorithm
+
+We treat each edge of the polygons as a vector, attaching them in polar order
+(attaching the tail of the next vector to the head of the previous vector). The
+resulting polygonal chain will be a polygon, which is the Minkowski sum of the given polygons.
```

Comment:
i think that the algorithm assumes that the vertices of `P` and of `Q` are sorted in counter-clockwise fashion. if it is the case then it's a good idea to say it in this paragraph.

Reply:
VPolygon's vertices are already sorted in a counterclockwise fashion (see the documentation for [VPolygon](https://juliareach.github.io/LazySets.jl/latest/lib/representations/#LazySets.VPolygon),  so I thought it was unnecessary, should I say it anyway?

Reply:
Well, yes you are right! :) So here i suggest to add a small remark as a reminder, in the end of your paragraph: "This algorithm assumes that the vertices of ``P`` and ``Q`` are sorted in counter-clockwise fashion and has linear complexity ``O(m+n)`` where ``m`` and ``n`` are the number of vertices of ``P`` and ``Q`` respectively."

Reply:
> it was unnecessary, should I say it anyway?

It goes in the direction of making the "Algorithm" notes self-contained so i think it's a good idea, even when the inputs are already `VPolygon`. (FWIW, `VPolygon` *can* be created with non-sorted vertices if you do `VPolygon(..., apply_convex_hull=false)`.)